tests/data/formats/SPDXJsonExample.json

```diff
@@ -1,223 +1,201 @@
 {
```

Comment:
Yes it is, this applies to all files in `data/` and `tests/data/formats`. I think this comes from the usages, the ones in `data/` are used as example files for the user and the other ones are used for tests. But I agree that we shouldn't keep duplicates. My approach would be to open another issue for that, since this implies more changes. Are you fine with that?

Reply:
#261

Reply:
I agree to make this a separate issue. And keeping the examples in some test resources folder sounds good to me.